app/controllers/admin/invitations_controller.rb

```diff
@@ -15,7 +15,7 @@ def create
       User.invite!(email: email,
                    name: name,
                    username: username,
-                   remote_profile_image_url: Users::ProfileImageGenerator.call,
+                   remote_profile_image_url: ::Users::ProfileImageGenerator.call,
```

Comment:
The tests were complaining about uninitialized `Admin::Users::ProfileImageGenerator`, so I had to specify this.